spec/features/show_peeps_spec.rb

```diff
@@ -0,0 +1,23 @@
+#feature 'Viewing test page' do
+  #scenario 'visiting the test page' do
+#  visit('/test')
+    #expect(page).to have_content "Test page"
+  #end
+#end
+require 'pg'
+feature 'Shows all peeps in database' do
+ scenario 'user presses a button and it shows peeps' do
+   visit ('/')
+   click_button('show all')
+   #expect(page.current_path).to eq('/')
+
```

Comment:
I don't you need the lines 10 and 11 here. 
The visit('/all') command will take the browser straight to that page and ignore the steps what has gone before.